packages/core/src/app/customer/StripeGuestForm.tsx

```diff
@@ -2,13 +2,14 @@ import { CustomerInitializeOptions, CustomerRequestOptions } from '@bigcommerce/
 import { FieldProps, FormikProps, withFormik } from 'formik';
 import React, { FunctionComponent, memo, ReactNode, useCallback, useEffect, useState } from 'react';
 
+import { CustomerSkeleton } from '@bigcommerce/checkout/ui';
```

Comment:
Recently, we got rid of the loading process for the customer step with https://github.com/bigcommerce/checkout-js/pull/1204, could you please check if it is also viable for `Stripe`?

Reply:
Hey @bc-peng, the problem is that it was a requirement made by Stripe since the loading times of the iframe are very long compared to the button, an apology

Reply:
OK, I was thinking removing `CustomerSkeleton` component.
In this case, we'll keep it.